pkg/server/skydb/pq/device.go

```diff
@@ -24,16 +24,46 @@ import (
 )
 
 func (c *conn) GetDevice(id string, device *skydb.Device) error {
-	builder := psql.Select("type", "token", "user_id", "last_registered_at").
+	builder := psql.Select("type", "token", "user_id", "topic", "last_registered_at").
 		From(c.tableName("_device")).
-		Where("id = ?", id)
+		Where("id = ? AND topic IS NOT NULL", id)
```

Comment:
After changing the interface of `APNSPusher`, it requires a topic to send push notification and the topic is referring to the device's one for keeping consistency of cert-based APNS and token-based APNS.

As the previous discussion, the old users need a migration to insert the device topic even though they keep using cert-based APNS.

That is why I set `topic IS NOT NULL` here.